### PR TITLE
Add tests with Jest

### PR DESCRIPTION
**Pull request by etrobert**
Note: The current config can not do snapshots with css because the `.css` is mocked as an empty module

### CODE REVIEW

File: src/scripts/MenuBar.test.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import React from "react";
+import noop from "lodash/noop";
+
+import MenuBar from "./MenuBar";
+
+describe("MenuBar", () => {
+  test("calls onLoad", () => {
```
Comment (justinline) [inline src/scripts/MenuBar.test.tsx:9]: question: Just curious why you didn't roll with `it`?

File: src/scripts/MenuBar.test.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import React from "react";
+import noop from "lodash/noop";
+
+import MenuBar from "./MenuBar";
+
+describe("MenuBar", () => {
```
Comment (justinline) [inline src/scripts/MenuBar.test.tsx:8]: suggestion: you could do `const noopProps = { onLoad: noop, onSave: noop, // etc }` and then

```
<Menubar {...noopProps} onLoad={mockOnLoad} />
```

to help keep focus on the thing being tested a little more

**Comment by justinline** [comment]
⭐ Finished! Yes tests!

File: src/scripts/MenuBar.test.tsx (hunk 2 above)
Comment (etrobert) [reply src/scripts/MenuBar.test.tsx:8]: 😱🤩

File: src/scripts/MenuBar.test.tsx (hunk 1 above)
Comment (etrobert) [reply src/scripts/MenuBar.test.tsx:9]: At work I kind of dislike that we use `it.each` that make way less sense than `test.each`

And it's not less readable to me, I think my mind is formatted to read this line as "test (that it) calls onLoad"

So there's no real reason for me to use the alias `it` as far as I can tell

Maybe you know of one?

Comment (etrobert) [reply src/scripts/MenuBar.test.tsx:9]: Overall it's a very non important and highly subjective difference 😄

Comment (justinline) [reply src/scripts/MenuBar.test.tsx:9]: I think it's more about the test output

with it: `module > should do ....`

with test: `module > calls onLoad`

the `should` reads just a bit more like an expectation. But it's subjective as you said :D

Comment (justinline) [reply src/scripts/MenuBar.test.tsx:9]: you could also write `test('should call onLoad)` though

Comment (etrobert) [reply src/scripts/MenuBar.test.tsx:9]: Where did you find that? 😮 

My test output is identical when running with `it` or `test`

And the Jest doc seems to imply that one is just an alias for the other ([link](https://jestjs.io/docs/api#testname-fn-timeout))

(`onLoad` uses `test` in the folowing screenshot)
image

Comment (etrobert) [reply src/scripts/MenuBar.test.tsx:9]: Ooooh nevermind I'm just very dumb

What you meant is that it changes the way we write our message 😅

Comment (justinline) [reply src/scripts/MenuBar.test.tsx:16]: I guess the issue is more about consistency. `it` kinda forces you to use `should` as the next word. Your `calls...` is fine for methods, but what about snapshot tests of accessibility tests? you cannot use `calls...` for all of them so you'll open it up to inconsistency in test descriptions in the current format

Comment (etrobert) [reply src/scripts/MenuBar.test.tsx:16]: After a quick googling I found that `it` is commonly used by other test frameworks so Jest may have added it for people not to have high switching cost 🤷 ([link](https://medium.com/weekly-webtips/my-discovery-journey-with-jest-test-vs-it-2cc7420a2cbe))

Comment (etrobert) [reply src/scripts/MenuBar.test.tsx:16]: To answer your expectation comment, when reading

```
> npm test
Menubar
  calls onLoad
```
It looks pretty explicit to me that it's an expectation 🤷

Comment (etrobert) [reply src/scripts/MenuBar.test.tsx:16]: If it boil down to personal preference, then:

I like `test` 😂

**Pull request merged**